contracts/src/v0.8/dev/ConfirmedOwnerWithProposal.sol

```diff
@@ -0,0 +1,113 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../interfaces/OwnableInterface.sol";
+
+/**
+ * @title The ConfirmedOwner contract
+ * @notice A contract with helpers for basic contract ownership.
+ */
+contract ConfirmedOwnerWithProposal is OwnableInterface {
+
+  address private s_owner;
+  address private s_pendingOwner;
+
+  event OwnershipTransferRequested(
+    address indexed from,
+    address indexed to
+  );
+  event OwnershipTransferred(
+    address indexed from,
+    address indexed to
+  );
+
+  constructor(
+    address owner,
```

Comment:
This shadows the [owner function declaration](https://github.com/smartcontractkit/chainlink/pull/4679/files#diff-40eb84cf18d8f41908f5bc1e0a4dd4c0855ea91bb509177cbb27fd49922a27d8R69). Does that have any implication?

Reply:
It shouldn't have any functional implications.

We could remove the warning by renaming this (and the same parameter in the v0.7 contract) to `newOwner` or `setOwner`, or something similar?

Reply:
Went with `newOwner`, as `ConfirmedOwner` actually uses `newOwner` too